Content.Server/Discord/Managers/LastMessageWebhookManager.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Server.Discord.Managers
+{
+    public class LastMessageWebhookManager
+    {
+        private readonly DiscordWebhook _discordWebhook = new DiscordWebhook();
```

Comment:
You cannot just create a new instance of this manager like this, you need to get it as a dependency.